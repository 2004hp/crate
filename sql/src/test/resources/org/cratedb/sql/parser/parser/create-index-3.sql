CREATE INDEX G1 ON "group_table" using fulltext(c1, c2, c3)