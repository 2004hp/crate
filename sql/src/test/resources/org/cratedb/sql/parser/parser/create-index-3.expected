org.cratedb.sql.parser.parser.CreateIndexNode@26fd68b1
name: g1
statementType: CREATE INDEX
unique: false
indexName: g1
tableName: group_table
indexMethod: fulltext
indexColumns: 
indexProperties: null
existenceCheck: NO_CONDITION
	
	org.cratedb.sql.parser.parser.IndexColumnList@4c986d3c

	[0]:	
	org.cratedb.sql.parser.parser.IndexColumn@2b2ba56d
	columnName: c1
	tableName: null
	ascending
	[1]:	
	org.cratedb.sql.parser.parser.IndexColumn@3e79ad3f
	columnName: c2
	tableName: null
	ascending
	[2]:	
	org.cratedb.sql.parser.parser.IndexColumn@77827284
	columnName: c3
	tableName: null
	ascending
